app/code/community/Algolia/Algoliasearch/Helper/Entity/Producthelper.php

```diff
@@ -645,55 +652,59 @@ protected function handlePrice(Mage_Catalog_Model_Product &$product, $sub_produc
                         $dashed_format = $this->formatPrice($min, false, $currency_code).' - '.$this->formatPrice($max,
                                 false, $currency_code);
 
-                        if (isset($customData[$field][$currency_code]['default_original_formated']) === false || $min <= $customData[$field][$currency_code]['default']) {
-                            $customData[$field][$currency_code]['default_formated'] = $dashed_format;
+                        if (isset($cdfc['default_original_formated']) === false || $min <= $cdfc['default']) {
+                            $cdfc['default_formated'] = $dashed_format;
 
                             //// Do not keep special price that is already taken into account in min max
-                            unset($customData['price']['special_from_date']);
```

Comment:
This code I'm not sure what it was trying to do, but I do not think there is a ['price'] key to $customData, because the prices under ['currency']